service/storage_service.hh

```diff
@@ -860,6 +860,8 @@ public:
     // Public for `reload_raft_topology_state` REST API.
     future<> topology_transition();
 
+    future<> update_service_levels_configuration();
```

Comment:
This function can only be called on shard 0, otherwise an assertion will be triggered.

This is an important precondition which should be stated as a part of the interface. Please put a comment above this declaration mentioning the precondition.